src/__test__/components/data-processing/CalculationConfigContainer.test.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { mount, configure } from 'enzyme';
+import preloadAll from 'jest-next-dynamic';
+import Adapter from 'enzyme-adapter-react-16';
+import { act } from 'react-dom/test-utils';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import { createStore, applyMiddleware } from 'redux';
+import { Alert, Button } from 'antd';
+import rootReducer from '../../../redux/reducers/index';
+
+import CalculationConfigContainer from '../../../components/data-processing/CalculationConfigContainer';
+import initialExperimentState from '../../../redux/reducers/experimentSettings/initialState';
+
+jest.mock('localforage');
+
+const sampleId = 'sample-WT';
+const sampleIds = ['sample-WT', 'sample-WT1', 'sample-KO'];
+const experimentId = 'e1234';
+const filterName = 'mitochondrialContent';
+
+const noData = {
+  experimentSettings: {
+    ...initialExperimentState,
+  },
+};
+
+const MockCalculationConfig = () => (
+  <>
+    hi
+  </>
+);
+describe('CalculationConfigContainer', () => {
+  let container = null;
+  let calculationConfig = null;
+  let store = null;
+  const onConfigChange = jest.fn();
+
+  beforeAll(async () => {
+    await preloadAll();
+  });
+
+  configure({ adapter: new Adapter() });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    store = createStore(rootReducer, noData, applyMiddleware(thunk));
+    container = mount(
+      <Provider store={store}>
+        <CalculationConfigContainer
+          experimentId={experimentId}
+          sampleId={sampleId}
+          sampleIds={sampleIds}
+          filterUuid={filterName}
+          plotType='unused'
+          onConfigChange={onConfigChange}
+        >
+          <MockCalculationConfig />
+        </CalculationConfigContainer>
+      </Provider>,
+    );
+    calculationConfig = container.find('MockCalculationConfig');
+  });
+  const setRadioButton = (value) => {
+    act(() => {
+      container.setProps({ value });
+      const radioButton = container.find(`input[value='${value}']`);
+      radioButton.simulate('change');
+    });
+  };
+  it('sets/resets a `disabled` property of contained components when auto/manual is set', () => {
+    const testRadioButton = (value) => {
+      setRadioButton(value);
+
+      // Useless... see comments below
+      container.update();
+      calculationConfig.update();
+
+      // The commented out test is the one I would like to pass,
+      // but I have not found a way to see useSelector do its magic
+
+      // expect(calculationConfig.props().disabled).toBe(value === 'automatic');
+      expect(store.getState().experimentSettings.processing[filterName][sampleId].auto).toBe(value === 'automatic');
+    };
+
+    testRadioButton('manual');
+    testRadioButton('automatic');
+  });
+  it.skip('displays a warning when the auto setting is changed', () => {
+    // TODO: works excuted on its own, fails as part of the test suite
```

Comment:
sounds like you are not clearing the mockup between running different tests. This test suite and potentially other test suites possibly need something like this:

```
 afterEach(() => {
    jest.clearAllMocks();
  });
 ```

Reply:
We already use it across tests in the UI

Reply:
There is a `jest.resetAllMocks()` in `beforeEach` already, but did not help

Reply:
did you try selectively disabling/enabling run of this test with a set of other unit tests in order to pin point which tests running together are a problem?